### PR TITLE
Static frontend complete 

### PR DESCRIPTION
**Pull request by Henry-C-Lo**
need to work on functionality and full screen button

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  render() {
+    const { galleryImages } = this.state;
+    return (
+      <div>
+      {/* <div style={{ background: `url(${pixelAlign})`, height:'100vh', backgroundRepeat: 'no-repeat' }}> */}
+        <h1>Gallery MaN gOd B</h1>
+        <div className={styles.app}>
+          <GalleryItemList galleryImages={galleryImages} />
```
Comment (songkenny) [inline client/src/components/App.jsx]: Recommend putting all of GalleryItemList into App

File: client/src/components/GalleryItemList.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <div className={styles.galleries_container}>
+      <div className={styles.galleryItemList}>
+        {galleryImages.map(image => (
+          <GalleryItem key={image._id} image={image} />
+        ))}
+      </div>
+      <Gallery />
+    </div>
+  )
+}
```
Comment (songkenny) [inline client/src/components/GalleryItemList.jsx]: Delete this component

**Review by songkenny — CHANGES_REQUESTED** [review]

File: coverage/clover.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ucb15erickao) [inline coverage/clover.xml:1]: Add coverage files to gitignore

File: tests/App.test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+import App from '../client/src/components/App';
+
+describe('App', () => {
+  test('should render', () => {
+    const wrapper = shallow(
+      <App />
+    )
+
+    expect(wrapper.exists()).toBeTruthy();
```
Comment (ucb15erickao) [inline tests/App.test.js:12]: I like this test. It doesn't need to work, it's okay.

File: yarn-error.log [hunk 5]
```diff
@@ -0,0 +1,84 @@
+Arguments:
```
Comment (ucb15erickao) [inline yarn-error.log:1]: git ignore

**Pull request merged**